@atoulme this seems related to your change:

```
running go unit test ./... + coverage in /home/circleci/project/exporter/splunkhecexporter
--- FAIL: Test_metricDataToSplunk (0.01s)
    --- FAIL: Test_metricDataToSplunk/double_gauge_empty_data_point (0.00s)
        metricdata_to_splunk_test.go:622: 
            	Error Trace:	metricdata_to_splunk_test.go:622
            	Error:      	Not equal: 
            	            	expected: 1
            	            	actual  : 0
            	Test:       	Test_metricDataToSplunk/double_gauge_empty_data_point
    --- FAIL: Test_metricDataToSplunk/int_gauge_empty_data_point (0.00s)
        metricdata_to_splunk_test.go:622: 
            	Error Trace:	metricdata_to_splunk_test.go:622
            	Error:      	Not equal: 
            	            	expected: 1
            	            	actual  : 0
            	Test:       	Test_metricDataToSplunk/int_gauge_empty_data_point
    --- FAIL: Test_metricDataToSplunk/int_histogram_empty_data_point (0.00s)
        metricdata_to_splunk_test.go:622: 
            	Error Trace:	metricdata_to_splunk_test.go:622
            	Error:      	Not equal: 
            	            	expected: 2
            	            	actual  : 0
            	Test:       	Test_metricDataToSplunk/int_histogram_empty_data_point
    --- FAIL: Test_metricDataToSplunk/double_histogram_empty_data_point (0.00s)
        metricdata_to_splunk_test.go:622: 
            	Error Trace:	metricdata_to_splunk_test.go:622
            	Error:      	Not equal: 
            	            	expected: 2
            	            	actual  : 0
            	Test:       	Test_metricDataToSplunk/double_histogram_empty_data_point
    --- FAIL: Test_metricDataToSplunk/int_sum_empty_data_point (0.00s)
        metricdata_to_splunk_test.go:622: 
            	Error Trace:	metricdata_to_splunk_test.go:622
            	Error:      	Not equal: 
            	            	expected: 1
            	            	actual  : 0
            	Test:       	Test_metricDataToSplunk/int_sum_empty_data_point
    --- FAIL: Test_metricDataToSplunk/double_sum_empty_data_point (0.00s)
        metricdata_to_splunk_test.go:622: 
            	Error Trace:	metricdata_to_splunk_test.go:622
            	Error:      	Not equal: 
            	            	expected: 1
            	            	actual  : 0
            	Test:       	Test_metricDataToSplunk/double_sum_empty_data_point
    --- FAIL: Test_metricDataToSplunk/double_histogram_no_upper_bound (0.00s)
        metricdata_to_splunk_test.go:622: 
            	Error Trace:	metricdata_to_splunk_test.go:622
            	Error:      	Not equal: 
            	            	expected: 2
            	            	actual  : 0
            	Test:       	Test_metricDataToSplunk/double_histogram_no_upper_bound
    --- FAIL: Test_metricDataToSplunk/int_histogram_no_upper_bound (0.00s)
        metricdata_to_splunk_test.go:622: 
            	Error Trace:	metricdata_to_splunk_test.go:622
            	Error:      	Not equal: 
            	            	expected: 2
            	            	actual  : 0
            	Test:       	Test_metricDataToSplunk/int_histogram_no_upper_bound
```

--- a/exporter/splunkhecexporter/metricdata_to_splunk_test.go
+++ b/exporter/splunkhecexporter/metricdata_to_splunk_test.go
@@ -619,7 +619,6 @@ func Test_metricDataToSplunk(t *testing.T) {
 			md := tt.metricsDataFn()
 			gotMetrics, gotNumDroppedTimeSeries := metricDataToSplunk(logger, md, &Config{})
 			assert.Equal(t, tt.wantNumDroppedTimeseries, gotNumDroppedTimeSeries)
-			assert.Equal(t, len(gotMetrics), len(tt.wantSplunkMetrics))
 			for i, want := range tt.wantSplunkMetrics {
 				assert.Equal(t, want, gotMetrics[i])
 			}

